classes/order/OrderCarrier.php

```diff
@@ -135,8 +135,8 @@ public function sendInTransitEmail($order)
         $orderLanguage = new Language((int) $orderLanguageId);
         $templateVars = [
             '{followup}' => str_replace('@', $this->tracking_number, $carrier->url),
-            '{firstname}' => $customer->firstname,
-            '{lastname}' => $customer->lastname,
+            '{firstname}' => !empty($customer->firstname) ? $customer->firstname : '',
```

Comment:
```suggestion
            '{firstname}' => $customer->firstname ?? '',
```
You are not consistent in the verification and callback to null in all of what you propose. Sometimes it is a `??` while at other times you use a tertiary accompanied by a `!empty()`. For more readability and consistency I would recommend using only `??`.

Reply:
Hi,

I've updated this file to use the ternary operator instead of ??, because PHPStan doesn't allow the null coalescing operator in this context. It doesn't accept that $customer->firstname (or lastname) might be null. If you prefer, I can apply the same ternary pattern consistently throughout the code.

Reply:
I think that more generally at the DB level there is a lack of consistency in the modification too. If we look at `id_gender` can now be null but `firstname` and `lastname` are necessarily a string (empty by default) why this technical choice?